python/cuml/fil/fil.pyx

```diff
@@ -537,7 +539,7 @@ class ForestInference(Base):
                           handle=None):
         """
         Creates a FIL model using the scikit-learn model passed to the
-        function.
+        function. This function requires Treelite 0.9 to be installed.
```

Comment:
All occurrences of 0.9 should be replaced with 0.90. I'm working on 0.90 release of Treelite now.